sdk/src/client/api/types.rs

```diff
@@ -169,7 +169,7 @@ impl TryFrom<RemainderDataDto> for RemainderData {
 impl From<&RemainderData> for RemainderDataDto {
```

Comment:
Maybe just make this ```impl From<RemainderData> for RemainderDataDto``` now, and remove the clones?

Reply:
Well but we can just remove the dto completely, no? https://github.com/iotaledger/iota-sdk/issues/1713

Reply:
Indeed!